Fix ad hoc booking

tochman opened the pull request:
Changes to the ad hoc booking endpoint after work on client. 


Ready for review!
@tochman @diraulo

tochman committed:
fixes ad hoc booking method in controller

tochman committed:
fixes to controller method tobook current 30 minutes slot
still WIP

tochman committed:
mixing with settings and refactor a bit of show template

diraulo commented on spec/requests/api/book_resource_endpoint_spec.rb:
```diff
@@ -21,6 +21,7 @@
                  headers: {'HTTP_ACCEPT': 'application/json'}}
       expected_response = eval(file_fixture('resource_ad_hoc_booking_response.txt').read)
       post "/api/resources/#{resource.uuid}/create_booking", payload
+      binding.pry
```
@tochman You forgot someone here

diraulo committed:
remove binding.pry statement in spec

tochman commented:
Damn! Good catch 😎

tochman committed:
fix for failing ad hoc booking response

tochman committed:
fixes helper spec
adds database cleaner configuration to rails helper
I wonder how we managed to do without it for so long...

tochman committed:
puts back needed variable to helper spec. Duh

tochman committed:
updates gems
